Portal 1.16 Documentation Update.mdx (#1041)
TykTechnologies/tyk-docs: tyk-stack/tyk-developer-portal/enterprise-developer-portal/api-access/approve-requests.mdx

To learn how to configure this flow visit [Access Flow Types](https://tyk.io/docs/portal/overview/concepts#Access-Flow-Types)

### Initial Steps (Both Flows):
1. From the **Catalogues** page, choose the API Product of interest and select **More info**
2. On the API Product detail page, decide which of the available Plans to subscribe to and select **Access with this Plan**
3. **The next steps depend on your portal's configured access flow**:

#### Direct Access Flow
When Direct Access Flow is enabled, you can request access to API Products immediately without using a shopping cart.

**Steps:**
- You'll be taken directly to the access request page with your selected product and plan pre-populated
- Select or create a Developer App to store your credentials
- Configure credentials (new or extend existing compatible credentials)
- Select **Continue** to submit your request

**Credential Compatibility Rules**

For existing credentials to be compatible, they must:
- Have the same Plan as the selected product
- Not already include the selected product
- Have a matching authentication type with the product

> 

Reviewer: ```suggestion

For more information, refer to this [guide](/tyk-developer-portal/direct-access-flow)
```

## Requesting Access to an API Product

> The Enterprise Developer Portal supports two different access flows for API Products. This is configured from the Admin Portal:

Reviewer 2: ```suggestion
The Developer Portal supports two different access flows for API Products. This is configured from the Admin Portal:
```
(moving away from referring to this as Enterprise Developer Portal -> just Developer Portal)

    - Not already include the selected product
    - Have a matching authentication type with the product

### Cart-Based Flow
When the Cart-Based Flow is enabled, you can add multiple API Products to a shopping cart before submitting a single access request.

#### Steps:
1. From the **Catalogues** page, choose the API Product of interest and select **More info**
2. On the API Product detail page, decide which of the available Plans to subscribe to and select **Access with this Plan**
3. The combination of API Product and Plan will be added to their *cart*
4. Repeat steps 1-3 for all the API Products you want to access
5. Go to the **Cart** using the icon in the top right of the screen
6. Review the selected API Products and Plan
    - All API Products must use the same Authentication method
    - A single subscription Plan will be selected
7. Select which Developer App to associate with the request
    - If approved, the Access Credentials will be stored in this Developer App
    - You can create a new App within the Cart, or select an existing App
8. Select **Submit request**

### After Submission
Regardless of which flow you use:

> -  Your access request will be submitted for approval (if required)

Reviewer 2: ```suggestion
- Your access request will be submitted for approval (if required)
```

    - The system sends a rejection notification email to the user
    - The email uses the template "reject" with a configurable subject

## Update Products and Plans of an Access Request

Starting from v1.16.0, the Tyk Developer Portal enables users to efficiently manage their access requests by adding or removing API Products and modifying subscription plans for existing credentials. This eliminates credential sprawl and provides a streamlined experience.

The following features can be accessed via "App" page, under each access credential.

#### Adding Products to Existing Access Request
Users can extend their existing credentials with additional API Products when compatibility requirements are met, avoiding the need to creating new credentials for additional API products.

**Compatibility Requirements**
Products can be added to existing credentials when they share:
- Same authentication method
- Same subscription plan
- Products not already included in the credential

When requesting access to an API Product, developers can choose to create a new credential or use one of the compatible existing credential.

> 

Reviewer: ```suggestion

For more information, refer to this [guide](/tyk-developer-portal/single-credentials-multiple-api-products)
```

    - The system sends a rejection notification email to the user
    - The email uses the template "reject" with a configurable subject

## Update Products and Plans of an Access Request

> The Tyk Developer Portal enables users to efficiently manage their access requests by adding or removing API Products and modifying subscription plans for existing credentials. This eliminates credential sprawl and provides a streamlined experience.

Reviewer 3: ```suggestion
Starting from v1.16.0, the Tyk Developer Portal enables users to efficiently manage their access requests by adding or removing API Products and modifying subscription plans for existing credentials. This eliminates credential sprawl and provides a streamlined experience.

The following features can be accessed via "App" page, under each access credential.
```

#### Adding Products to Existing Access Request

> Users can extend their existing credentials with additional API Products when compatibility requirements are met, avoiding the need to manage multiple credentials for the same application.

Reviewer 3: ```suggestion
Users can extend their existing credentials with additional API Products when compatibility requirements are met, avoiding the need to creating new credentials for additional API products.
```

**Compatibility Requirements**
Products can be added to existing credentials when they share:
- Same authentication method
- Same subscription plan
- Products not already included in the credential

#### Access Request Flows
**Direct Access Flow**: When requesting access to an API Product, developers can choose to:
 - **Generate new credentials**: Creates a separate credential set
 - **Use existing credentials**: Adds the product to compatible existing credentials
**Cart-Based Flow**: During checkout, the system identifies compatible credentials and allows developers to:
 - Extend existing credentials with new products

>  - Create new credentials when no compatible options exist

Reviewer 3: ```suggestion
When requesting access to an API Product, developers can choose to create a new credential or use one of the compatible existing credential.
```

#### Removing Products from Access Requests

> Developers can remove individual API Products from credentials that contain multiple products, maintaining access to remaining products while revoking access to unwanted ones. This provides granular control over application permissions without affecting the entire credential.

Reviewer 3: ```suggestion
Users can remove individual API Products from credentials that contain multiple products, maintaining access to remaining products while revoking access to unwanted ones. This provides granular control over application permissions without affecting the entire credential.
```

#### Plan Management

> Portal users can update subscription plans for existing access requests without generating new credentials. Plan modifications maintain existing product associations where compatibility allows.

Reviewer 3: ```suggestion
Users can update subscription plans for existing access requests without generating new credentials. Plan modifications maintain existing product associations where compatibility allows.
```

The following features can be accessed via "App" page, under each access credential.

#### Adding Products to Existing Access Request
Users can extend their existing credentials with additional API Products when compatibility requirements are met, avoiding the need to creating new credentials for additional API products.

**Compatibility Requirements**
Products can be added to existing credentials when they share:
- Same authentication method
- Same subscription plan
- Products not already included in the credential

When requesting access to an API Product, developers can choose to create a new credential or use one of the compatible existing credential.

For more information, refer to this [guide](/tyk-developer-portal/single-credentials-multiple-api-products)
#### Removing Products from Access Requests
Users can remove individual API Products from credentials that contain multiple products, maintaining access to remaining products while revoking access to unwanted ones. This provides granular control over application permissions without affecting the entire credential.

#### Plan Management

> Users can update subscription plans for existing access requests without generating new credentials. Plan modifications maintain existing product associations where compatibility allows.

Reviewer 3: ```suggestion
Users can update subscription plans for existing access requests without generating new credentials. Plan modifications maintain existing product associations where compatibility allows.

Currently, Plan management is available to OAuth 2.0 access requests only.
```
